### PR TITLE
Added GitHub integration to alabaster theme

### PR DESCRIPTION
**Pull request by ehsteve**

### CODE REVIEW

**Comment by ddasilva** [comment]
I kind of think having the stars count like we used to have makes more sense than the watch count. It gives a similar endorsement metric like the "used by" section. I personally never watch repositories so maybe I'm biased. What do you think?

image

**Comment by ehsteve** [comment]
I do agree with I don't understand the [documentation](https://alabaster.readthedocs.io/en/latest/customization.html#theme-options). They mention `github_type` and `github_count` as "options" but not sure how to toggle them and, in any case, they don't seem to be what you want. Should I get rid of it then?

**Comment by ddasilva** [comment]
Can we add back the code we had before to display the Github stars? I pulled it out from an old commit and pasted it below

````html
<iframe src="https://ghbtns.com/github-btn.html?user=ddasilva&repo=ccsdspy&type=star&count=true&size=large" frameborder="0" scrolling="0" width="160px" height="30px"></iframe>

**Comment by ehsteve** [comment]
Done. Also added a link out to CCSDS. Not sure if you want it but seemed nice to me to provide for users.

**Comment by ddasilva** [comment]
> Done. Also added a link out to CCSDS. Not sure if you want it but seemed nice to me to provide for users.

Github stars look good! Linking out to CCSDS is welcome

File: docs/conf.py [hunk 1]
```diff
@@ -99,21 +100,29 @@
...
 html_theme_options = {
-    'show_related': True
+    "description": "A Python package for reading CCSDS packet data.",
+    "show_related": True,
+    "github_banner": True,
+    "github_button": False,
+    "github_user": "ddasilva",
+    "github_repo": "ccsdspy",
+    "extra_nav_links": {
+        "ccsds blue books": "https://public.ccsds.org/publications/bluebooks.aspx"
```
Comment (ddasilva) [inline docs/conf.py:117]: can you style the capitalization "CCSDS Blue Books"?

Comment (ehsteve) [inline docs/conf.py:117]:
```suggestion
        "CCSDS blue books": "https://public.ccsds.org/publications/bluebooks.aspx"
```

Comment (ehsteve) [reply docs/conf.py]: done

**Comment by ddasilva** [comment]
Production site has been rebuilt:
https://ccsdspy.readthedocs.io/en/latest/

**Pull request merged**